File: SoftverskaRealizacijaBackend/SoftverskaRealizacijaBackend/Dto/ClientDto.cs
using static SoftverskaRealizacijaBackend.Models.Enumerations;

namespace SoftverskaRealizacijaBackend.Dto
{
    public class ClientDto
    {
        public int Id { get; set; }
        public string FullName { get; set; }
        public TipKorisnika TipKorisnika { get; set; }
        public string Email { get; set; }
        public string Password { get; set; }
    }
}

File: softverskarealizacijafront/src/Models/ClientDto.js
export default class ClientDto{
    constructor(data){
        this.id = data.id;
        this.fullName = data.fullName;
        this.email = data.email;
        this.password = data.password;
        this.ime = data.ime;
        
        this.tipKorisnika = data.tipKorisnika;
           
    }
}